hack/verify-util-pkg.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"/pkg/util
+
+find_files() {
+  find . -maxdepth 1 ! -type d -not \( \
+      \( \
+        -wholename './BUILD' \
+        -o -wholename './OWNERS' \
+        -o -wholename './doc.go' \
+      \) -prune \
+    \)
+}
+
+util_files=($(find_files | while read file; do
+  file=${file#./}
+  echo "pkg/util/${file}"
+done))
+
+if [ ${#util_files[@]} -gt 0 ]; then
+  exec 1>&2
```

Comment:
because it's only the echo that needs this, the `1>&2` can just be tacked on to that one command